src/patternfly/components/TextInputGroup/text-input-group.scss

```diff
@@ -103,9 +103,10 @@
     }
   }
 
-  .pf-c-chip-group,
-  .pf-c-chip-group__list {
-    flex-wrap: nowrap;
+  .pf-c-chip-group li,
```

Comment:
Is there an `li` in chip group that's outside of `__list`? Curious why you need this and the line below it.

Also could you target `.pf-c-chip-group__list-item` instead of `li`?

And now that we're allowing these to wrap, I'm not sure why we need to update `min-width` - I wouldn't think they would shrink if they can wrap instead. Disabling this `min-width: fit-content` style in dev tools doesn't break anything as far as I can tell, though I'm probably missing something 🤓